test/src/models/CourtDeployer.js

```diff
@@ -0,0 +1,160 @@
+const path = require('path')
+const Config = require('../../../config/court')
+const Environment = require('../../../src/models/Environment')
+const CourtDeployer = require('../../../src/models/CourtDeployer')
+
+const { assertBn } = require('../../helpers/assertBn')
+const { MAX_UINT64 } = require('../../../src/helpers/numbers')
+
+contract('CourtDeployer command', ([sender]) => {
```

Comment:
Deployment tests 🤩

Love that we can run them against real nodes and not just ganache